src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -961,10 +972,15 @@
         },
 
         "module": {
-          "description": "The name of the module that contains the code that is executing.",
+          "description": "The name of the module that contains the code for this stack frame.",
           "type": "string"
         },
 
+        "threadId": {
+          "description": "The thread identifier of the stack frame.",
+          "type": "integer"
```

Comment:
For Module, you wrote "for this stack frame"; here, you write "of the stack frame". I think the uses of "for" and "of" are fine. I'd prefer "... of this stack frame" for more consistency.